### PR TITLE
Add an icon for Mapbox

### PR DESCRIPTION
**Pull request by jeandeaual**
The icon comes from their official brand guidelines: https://www.mapbox.com/about/press/brand-guidelines/

### CODE REVIEW

File: app/src/main/res/drawable/thumb_mapbox.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="800dp"
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_mapbox.xml]: This is too big for android to render reliably, we try to keep under 240

Comment (jeandeaual) [reply app/src/main/res/drawable/thumb_mapbox.xml]: I reduced the width and height without changing the viewportWidth and viewportHeight. Is that enough?

**Comment by flocke** [comment]
We currently have two PRs for the MapBox icon. I'm gonna close this one since I prefer the version in #238 because it's only the icon and not the full logo and thus fits the aspect ration of the thumbnails in andOTP better.
Nevertheless thank you for your PR!

**Comment by jeandeaual** [comment]
I agree that the logo from #238 looks better. I used the full logo instead of just the icon since the Mapbox brand guidelines mention the following:

> The Mapbox logo is a combination of the wordmark with the icon. Please always use the wordmark and icon together. Refrain from using either the wordmark or the icon on its own.

I'm not sure if this matters here.
Anyway, thanks for the review!

**Pull request closed without merging**